torchtext/experimental/datasets/text_classification.py

```diff
@@ -1,93 +1,479 @@
-import logging
 import torch
-import io
-from torchtext.utils import download_from_url, extract_archive
-from torchtext.data.utils import ngrams_iterator
 from torchtext.data.utils import get_tokenizer
 from torchtext.vocab import build_vocab_from_iterator
-from torchtext.vocab import Vocab
-from torchtext.datasets import TextClassificationDataset
+from torchtext.experimental.datasets.raw import AG_NEWS as RawAG_NEWS
```

Comment:
Instead of this should be able to just import "raw" and then index into this using the name of the dataset you're calling this from.